docs/design/ocs-to-ocs-api.md

```diff
@@ -0,0 +1,96 @@
+# FEATURE: OCS to OCS API changes
+
+## Summary
+Allow customers to share OCS storage across multiple OCP clusters. Each
+consumer OCP cluster will have OCS deployed that connects to the provider
+cluster.
+
+## Motivation
+This feature gives flexibility to customers and lets them share storage and
+better utilize their OpenShift clusters.
```

Comment:
It is more cost effective, especially for cases when the storage capacity needed by the OCP cluster is low.